test/integration/controllermanager/shoot/maintenance/maintenance_test.go

```diff
@@ -342,6 +342,12 @@ var _ = Describe("Shoot Maintenance controller tests", func() {
 				g.Expect(testClient.Get(ctx, client.ObjectKeyFromObject(shoot), shoot)).To(Succeed())
 				g.Expect(*shoot.Spec.Provider.Workers[0].Machine.Image).To(Equal(gardencorev1beta1.ShootMachineImage{Name: highestAMD64MachineImage.Name, Version: highestAMD64MachineImage.Version}))
 				g.Expect(*shoot.Spec.Provider.Workers[1].Machine.Image).To(Equal(gardencorev1beta1.ShootMachineImage{Name: highestARM64MachineImage.Name, Version: highestARM64MachineImage.Version}))
+				g.Expect(shoot.Status.LastMaintenances).ToNot(BeNil())
```

Comment:
```suggestion
				g.Expect(shoot.Status.LastMaintenances).NotTo(BeEmpty())
```
since it's a slice. same for all other occurences.